packages/plugins/graphql/server/services/utils/config.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+/**
+ * GraphQL config helper with consistent defaults values
+ */
+module.exports = ({ strapi }) => {
```

Comment:
why not put that in the config default values instead ?

Reply:
It doesn't look like it's necessary

Reply:
didn't know it existed, will do it